modules/install-ibm-cloud-configuring-the-install-config-file.adoc

```diff
@@ -0,0 +1,79 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-install-config-file_{context}"]
+= Configuring the install-config.yaml file
+
+This procedure is a complement to the xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-the-install-config-file_ipi-install-configuration-files[Configuring the install-config.yaml file] procedure in the "Setting up the environment for an OpenShift installation" section of the "Deploying installer-provisioned clusters on bare metal" documentation.
+
+The `install-config.yaml` file requires some additional details. Most of the information is teaching the installer and the resulting cluster enough about the available IBM Cloud&#174; hardware so that it is able to fully manage it. The material difference between installing on bare metal and installing on IBM Cloud is that you must explicitly set the privilege level for IPMI in the BMC section of the `install-config.yaml` file.
+
+.Procedure
+
+. Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
++
+[source,yaml]
+----
+  apiVersion: v1
+  baseDomain: <domain>
+  metadata:
+    name: <cluster-name>
+  networking:
+    machineCIDR: <public-cidr>
+    networkType: OVNKubernetesThis document describes configuring Dnsmasq on the public subnet.
```

Comment:
Is there a full stop missing or is this out of place "OVNKubernetesThis document describes configuring Dnsmasq on the public subnet."